tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/OnBoardMember.kt

```diff
@@ -195,7 +195,41 @@ class OnBoardMember : Runnable, BaseOnboard() {
 
     override val registrationContext by lazy {
         val preAuth = preAuthToken?.let { mapOf("corda.auth.token" to it) } ?: emptyMap()
+        
+        if (customProperties.size > 100) {
+            throw IllegalArgumentException("Cannot specify more than 100 key-value pairs")
+        }
+        
         val notaryServiceName = customProperties["corda.notary.service.name"]
+      
+        customProperties = customProperties.filterKeys { it != "corda.notary.service.name" }
+        
+        val extProperties = customProperties.filter {
```

Comment:
I think something like this should be sufficient here -
```suggestion
        val extProperties = customProperties.filterKeys { it.startsWith("$CUSTOM_KEY_PREFIX.") }
```

Reply:
Should I use a companion object for specifying the constants in the same class? Or should I just create a separate constants class for this, the question is that I am not sure if we are going to be reusing these values elsewhere in the plugin classes.

Reply:
You can reuse the ones from `MemberInfoExtension`, they are already defined there